packages/hop-node/src/watchers/HealthCheckWatcher.ts

```diff
@@ -129,6 +133,22 @@ type InvalidBondWithdrawal = {
   timestamp: number
 }
 
+type UnrelayedTransfers = {
+  transactionHash: string
+  token: string
+  recipient: string
+  destinationChainId: number
+  amount: BigNumber
+  relayer: string
+  relayerFee: BigNumber
+}
+
+type UnsetTransferRoots = {
+  transferRootHash: string
+  totalAmount: BigNumber
```

Comment:
If these entities are part of the health checker json output then the amount types should be `string` instead of `BigNumber`